blocks/Cairo/include/cinder/cairo/Cairo.h

```diff
@@ -44,47 +44,8 @@
 #include <vector>
 #include <iomanip>
 
-// Forward declarations used by our cairo wrappers 
```

Comment:
I didn't see the reason for this, but if there actually is one we can restore the code and just add an IF macro. During the linux build it would create conflicts with cairo.h

Reply:
Are the conflicts because the forward declarations need to be updated? I haven't worked with the Cairo.h wrapper personally, but my guess is that this was avoiding the heavy `#include <cairo.h>` so as cinder end user apps don't have to bring all of those symbols into their files. In general, we try not to include heavy system headers from within cinder headers.

Reply:
Let's talk about it in #2136 :)